src/paymentservice/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "paymentservice",
-  "version": "1.4.0",
+  "version": "1.12.0",
```

Comment:
Do we need this entry here? Seems like an extra place we need to maintain a version number that I would prefer not to.